src/azul/auth.py

```diff
@@ -12,6 +24,42 @@ def identity(self) -> str:
         """
         raise NotImplementedError
 
+    _cls_field = '_cls'
```

Comment:
Don't you have to type this as ClassVar to prevent auto_attribs from catching it?